studio-docs/source/github-integration.md

```diff
@@ -23,14 +23,14 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: cimg/node:16.0.0
 
     steps:
       - checkout
 
       - run: npm install
       # CircleCI needs global installs to be sudo
-      - run: sudo npm install --global apollo
+      - run: sudo npm install --global @apollo/rover
```

Comment:
I think we were previously fighting peer dependencies and overlapping `graphql` versions, but since we're done with that with the introduction of Rover, I think in a future / soon change to these instructions it would be advisable for users to directly install `@apollo/rover` in their application's `devDependencies`, and then merely call `npx rover` here from their project (after the `npm install`).

That would provide them a pinned version and reproducible build dependencies by avoiding always just installing the _`latest`_ version of `@apollo/rover`.  (Also, presumably, their developers would appreciate having `npx rover` available to them.  

I wonder where would be a good place to track this work?  Would this leverage an installation instruction best-practice elsewhere?  Or would it just make the suggestion here itself?

Reply:
Good idea! I'll make "general CI tidying" a near-future focus for nuances like this.